server/config-example.js

```diff
@@ -4,7 +4,7 @@ var config = {
   couchDbPort: '5984',
   couchDbUseSsl: false,
   couchDbChangesSince: 'now',
-  couchAdminUser: 'couchadmin',
+  couchAdminUser: 'hradmin',
```

Comment:
Can this be added to the README. The difference between `hradmin` and `couchadmin` really confused me (I didn't even know there was a `couchadmin`) till I saw this comment.